docs/partials/deploy-hyperlane/_deploy-contracts.mdx

```diff
@@ -24,25 +24,20 @@ npm install -g @hyperlane-xyz/cli
 
 <CliChainsPartial />
 
-#### ISM
+#### Core deployment with Multisig ISM
 
 Now the CLI will know how to interact with all your chains, but it will also need to know how to configure your Interchain Security Module (ISM).
 
-To create a multisig ISM configs, you can define it manually using JSON or YAML (see [example config here](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/v3/typescript/cli/examples/multisig-ism.yaml)), or create it with the following command:
+To initialize a core config with multisig ISM config, you can define it manually using JSON or YAML (see [example config here](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/cli/examples/core-config.yaml)), or create it with the following command:
```

Comment:
nit: I know the previous URL was also relative, but I think it's good practice to use permalinks instead so we always point to the same commit. This prevents having broken links when things get moved around